posts/2022-02-24-Rust-1.59.0.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: "Announcing Rust 1.59.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.59.0.
+Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.59.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.59.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1590-2022-02-24
+
+## What's in 1.59.0 stable
+
+### Inline assembly
+
+The Rust language now supports inline assembly. This enables many applications
+of Rust that need very low-level control over their execution, or access to
+specialized machine instructions.
+
+When compiling for x86 and x86-64 targets, for instance, you can now write:
```

Comment:
```suggestion
When compiling for x86-64 targets, for instance, you can now write:
```

The asm as written doesn't work on x86-32 because it wants a 64-bit register. Changing the `u64` to a `u32` would work on x86-32, but require some additional distracting changes to work on x86-64 (to use an `e` register rather than an `r` register). So, just narrow the example to x86-64.